packages/generator/src/helper.ts

```diff
@@ -294,16 +304,28 @@ const drawInputByTextSchema = async (arg: {
         pdfFontValue.widthOfTextAtSize(testString, size) + (testString.length - 1) * characterSpacing;
       return width <= testStringWidth;
     };
-    const splitedLines = getSplittedLines(inputLine, isOverEval);
-    const drawLine = (splitedLine: string, splitedLineIndex: number) => {
-      const textWidth =
-        pdfFontValue.widthOfTextAtSize(splitedLine, size) +
-        (splitedLine.length - 1) * characterSpacing;
-      page.drawText(splitedLine, {
+    const splitLines = getSplittedLines(inputLine, isOverEval);
+
+    const drawLine = (line: string, lineIndex: number) => {
+      const textWidth = pdfFontValue.widthOfTextAtSize(line, size) + (line.length - 1) * characterSpacing;
+      const fallbackFont = getDefaultFont();
```

Comment:
move to the outside of for loop.
this code performance is bad.
```
      const fallbackFont = getDefaultFont();
      let schemaFontData = fallbackFont[DEFAULT_FONT_NAME].data;

      if (templateSchema.fontName) {
        schemaFontData = font[templateSchema.fontName].data;
      }

      const fontkitFont = fontkit.create(Buffer.from(schemaFontData as ArrayBuffer));
```

Reply:
I have update this file to move the above further up the `helper.ts` file. This will be visible in my next commit 👍